### DISCUSSION TITLE
4.2.0

### DISCUSSION THREAD

**Discussion by nalexandru** [nalexandru/BlenderCompat#94] [Announcements]
Blender 4.2.0, commit [a51f293548ad](https://projects.blender.org/blender/blender/commit/a51f293548ad).
Build environment:

    Visual Studio 2019, version 16.11.37
    Windows SDK 10.0.22621
    NVIDIA CUDA 11.8
    NVIDIA OptiX 7.3

AMD HIP and SM3.0 kernel binaries copied from the official release.

<hr /><em>This discussion was created from the release <a href='https://github.com/nalexandru/BlenderCompat/releases/tag/4.2.0'>4.2.0</a>.</em>

**Comment by patientAtGithub** [comment]
Thank you! Thank you! Thank you!

**Comment by kaiwas** [comment]
> Blender 4.2.0

Hello.
Should we expect version 4.2.1?
Thanks for your work!

**Reply by nalexandru** [reply]
I generally try to build new versions within a few days from the official release.
The last months have been quite busy which is why I missed 4.2.1.

**Reply by kaiwas** [reply]
> I missed 4.2.1.

No problem. You need to proceed from your capabilities.
Thanks for 4.2.2 and for your work in general!

**Comment by patientAtGithub** [comment]
Thank you again Alexandru.
As always, very very much appreciated.
